control-plane-operator/controllers/hostedcontrolplane/etcd/params.go

```diff
@@ -10,60 +10,60 @@ import (
 )
 
 type EtcdParams struct {
-	ClusterVersion            string
-	EtcdOperatorImage         string
-	OwnerRef                  config.OwnerRef `json:"ownerRef"`
-	OperatorDeploymentConfig  config.DeploymentConfig
-	EtcdDeploymentConfig      config.DeploymentConfig
-	PersistentVolumeClaimSpec *corev1.PersistentVolumeClaimSpec `json:"persistentVolumeClaimSpec"`
+	EtcdImage string
+
+	OwnerRef         config.OwnerRef `json:"ownerRef"`
+	DeploymentConfig config.DeploymentConfig
+
+	StorageSpec hyperv1.ManagedEtcdStorageSpec
 }
 
-var etcdLabels = map[string]string{
+var etcdPodSelector = map[string]string{
```

Comment:
Remember how we just talked about package-level vars? :P
```suggestion
func etcdPodSelector() map[string]string(
    return map[string]string{"app":"etcd"}
)
```

Reply:
Fixed